### PR TITLE
Support multiple creation of views & Fix illegal view creation

### PR DESCRIPTION
**Pull request by citrusreticulata**
### Support multiple creation of views
```
IoTDB> CREATE TIMESERIES root.db.d01.s01 INT32 encoding=RLE compression=SNAPPY;
Msg: The statement is executed successfully.
IoTDB> CREATE TIMESERIES root.db.d01.s02 INT32 encoding=RLE compression=SNAPPY;
Msg: The statement is executed successfully.
IoTDB> CREATE TIMESERIES root.db.d02.s01 INT32 encoding=RLE compression=SNAPPY;
Msg: The statement is executed successfully.
IoTDB> CREATE TIMESERIES root.db.d02.s02 INT32 encoding=RLE compression=SNAPPY;
Msg: The statement is executed successfully.
IoTDB>
IoTDB> CREATE VIEW root.multi_view.all_in_one(${2}_${3}) AS SELECT * FROM root.db.**;
Msg: The statement is executed successfully.
IoTDB>
IoTDB> CREATE VIEW root.copy_view.${2}(${3}) AS SELECT * FROM root.db.**;
Msg: The statement is executed successfully.
IoTDB>
IoTDB> SHOW VIEW;
+----------------------------------+---------------+--------+----+----------+--------+---------------+
|                        Timeseries|       Database|DataType|Tags|Attributes|ViewType|         Source|
+----------------------------------+---------------+--------+----+----------+--------+---------------+
|root.multi_view.all_in_one.d02_s02|root.multi_view|   INT32|null|      null| logical|root.db.d02.s02|
|root.multi_view.all_in_one.d01_s02|root.multi_view|   INT32|null|      null| logical|root.db.d01.s02|
|root.multi_view.all_in_one.d01_s01|root.multi_view|   INT32|null|      null| logical|root.db.d01.s01|
|root.multi_view.all_in_one.d02_s01|root.multi_view|   INT32|null|      null| logical|root.db.d02.s01|
|            root.copy_view.d01.s02| root.copy_view|   INT32|null|      null| logical|root.db.d01.s02|
|            root.copy_view.d01.s01| root.copy_view|   INT32|null|      null| logical|root.db.d01.s01|
|            root.copy_view.d02.s02| root.copy_view|   INT32|null|      null| logical|root.db.d02.s02|
|            root.copy_view.d02.s01| root.copy_view|   INT32|null|      null| logical|root.db.d02.s01|
+----------------------------------+---------------+--------+----+----------+--------+---------------+
Total line number = 8
It costs 0.033s
IoTDB>
```

### fix illegal timeseries creation caused by select into
```
IoTDB> SELECT s01, s02 into root.db.myinto.d01(${3}_$abc) FROM root.db.d02;
Msg: 701: Parsed node name s01_$abc is illegal, unquoted node name can only consist of digits, characters and underscore, or start or end with wildcard
IoTDB>
```

### views created by aggregation function is illegal now.
```
IoTDB> CREATE VIEW root.cal_view(agg_avg1, agg_avg2) AS SELECT AVG(s01)+1 FROM root.db.d01, root.db.d02;
Msg: 300: View unsupported, because: This view contains aggregation function(s) named [avg(root.db.d01.s01)]
IoTDB>
```

### Fixed a bug where using duplicate paths can also create views;
```
IoTDB> CREATE VIEW root.copy_view(a, a) AS SELECT s01, s02 FROM root.db.d01;
Msg: 509: Path [root.copy_view.a] is redundant in target paths.
IoTDB>
IoTDB> CREATE VIEW root.copy_view.abc, root.copy_view.abc AS SELECT s01, s02 FROM root.db.d01;
Msg: 509: Path [root.copy_view.abc] is redundant in target paths.
IoTDB>
```

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/metadata/view/ViewPathType.java [hunk 1]
```diff
@@ -20,7 +20,13 @@
...
+  /** Example: root.db.d01.s01, root.db.d02.s01, root.db.d02.s02 */
   FULL_PATH_LIST,
+  /** Example: root.db(d01.s01, d02.s02, status) */
   PATHS_GROUP,
-  QUERY_STATEMENT
+  /** Example: SELECT s01, s02 FROM root.db.*; */
+  QUERY_STATEMENT,
+
+  /** Example: root.db.view(${2}_temperature) */
+  INTO_ITEM,
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/metadata/view/ViewPathType.java:31]: Rename to ```BATCH_GENERATION```

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/metadata/view/CreateLogicalViewStatement.java [hunk 2]
```diff
@@ -21,22 +21,30 @@
...
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 /** CREATE LOGICAL VIEW statement. */
 public class CreateLogicalViewStatement extends Statement {
 
   // the paths of this view
   private ViewPaths targetPaths;
+  private IntoItem intoItem;
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/metadata/view/CreateLogicalViewStatement.java:47]: Rename the variable name to ```batchGenerationItem```, which is implemented based on IntoItem.

File: server/src/main/java/org/apache/iotdb/db/metadata/view/ViewPathType.java (hunk 1 above)
Comment (citrusreticulata) [reply server/src/main/java/org/apache/iotdb/db/metadata/view/ViewPathType.java]: resolved

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/metadata/view/CreateLogicalViewStatement.java (hunk 2 above)
Comment (citrusreticulata) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/metadata/view/CreateLogicalViewStatement.java]: resolved

**Pull request merged**